forge-gui/res/cardsfolder/upcoming/oblivious_bookworm.txt

```diff
@@ -0,0 +1,14 @@
+Name:Oblivious Bookworm
+ManaCost:G U
+Types:Creature Human Wizard
+PT:2/3
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | OptionalDecider$ You | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ At the beginning of your end step, you may draw a card. If you do, discard a card unless a permanent entered the battlefield face down under your control this turn or you turned a permanent face up this turn.
+SVar:TrigDraw:DB$ Draw | SubAbility$ DBDiscard
```

Comment:
write with cost?

Reply:
hmn, actually it looks like it checks the condition first
so you should just fix it with normal Remembering
(which probably means the static needs to use Imprinted to avoid collision 🤔)

Reply:
Hmm this will need some extra logic I think. For the discard to function we needs draw = 1 and condition=0

Reply:
seems ok now?

Reply:
I think so yeah